test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Negotiate.cs

```diff
@@ -380,6 +382,37 @@ public async Task StartSkipsOverTransportsThatDoNotSupportTheRequredTransferForm
                     });
             }
 
+            [Fact]
+            public async Task NegotiateThatReturnsErrorThrowsFromStart()
+            {
+                bool ExpectedError(WriteContext writeContext)
+                {
+                    return writeContext.LoggerName == typeof(HttpConnection).FullName &&
+                        writeContext.EventId.Name == "ErrorWithNegotiation";
+                }
+
+                var testHttpHandler = new TestHttpMessageHandler(autoNegotiate: false);
+                testHttpHandler.OnNegotiate((request, cancellationToken) =>
+                {
+                    return ResponseUtils.CreateResponse(HttpStatusCode.OK,
+                            JsonConvert.SerializeObject(new
+                            {
+                                error = "Negotiate failed."
```

Comment:
nit: Change this error message to something that isn't about negotiation. This seems like something that could be ambiguous